platform/android/app/src/test/kotlin/coop/polypoly/polypod/updatenotification/UpdateNotificationTest.kt

```diff
@@ -12,6 +13,7 @@ import org.robolectric.annotation.LooperMode
 @LooperMode(LooperMode.Mode.PAUSED)
 @RunWith(AndroidJUnit4::class)
 @Config(sdk = [Config.OLDEST_SDK])
+@Ignore("Not working on GitHub Actions")
```

Comment:
Where did it fail? I can't find it...

Reply:
Was a flapper, removed it again in the mean time.